crates/key-value/src/host_component.rs

```diff
@@ -1,17 +1,37 @@
-use crate::{Impl, KeyValueDispatch};
-use spin_app::DynamicHostComponent;
+use crate::{KeyValueDispatch, StoreManager};
+use spin_app::{AppComponent, DynamicHostComponent};
 use spin_core::HostComponent;
-use std::{collections::HashMap, sync::Arc};
+use std::sync::Arc;
 
 pub const KEY_VALUE_STORES_METADATA_KEY: &str = "key_value_stores";
```

Comment:
I think it may make sense to make this const (`KEY_VALUE_STORES_METADATA_KEY`) public outside the crate